### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by quasicomputational** [quchen/prettyprinter#81]
The current release of `prettyprinter` doesn't work with GHC 8.8. This is already fixed in HEAD, but it'd be nice to have a release to unblock downstreams on upgrading. Can one be made, please?

**Comment by quasicomputational** [comment]
Ah - `prettyprinter` is actually fine. I was getting confused by some stray bounds keeping me from using 1.3.0.
